connection/auth.go

```diff
@@ -115,6 +115,11 @@ func loginOIDC(oidcConfig *config.OIDCConfig, manualLogin bool) (*claims, error)
 		return nil, fmt.Errorf("login failed: %w", err)
 	}
 
```

Comment:
It would be great if you could add a link to the documentation that says that this is the standardized way of refreshing tokens.